### DISCUSSION TITLE
about global access write

### DISCUSSION THREAD

**Discussion by HCSR-Lab** [accel-sim/accel-sim-framework#224] [General] [answered]
I am using RTX2060 and **a global access write failure occurred** in the L2 cache. What will happen afterwards? I did some printing and did not find the next step to proceed with the write operation.
In addition, when the instruction performs a global write operation, multiple access L2 caches are all Misses and have the same address.
Under normal circumstances, I believe that the relative data will be refilled and global writes will be performed, but I did not do so. Why is this?
[gpgpu-sim/l2cache.cc:535](https://github.com/accel-sim/gpgpu-sim_distribution/blob/dev/src/gpgpu-sim/l2cache.cc)

**Comment by JRPan** [comment, marked as answer]
Double-check your config file. It should be LAZY_FETCH_ON_READ. For more you can read: Jouppi, Norman P. "Cache write policies and performance". ISCA 93.

**Reply by HCSR-Lab** [reply]
Okay, thank you for your reply. Through your reply, I already know what happened.
